WalletWasabi.Gui/Behaviors/PasteAddressOnClickBehavior.cs

```diff
@@ -96,25 +125,47 @@ protected override void OnAttached()
 				AssociatedObject.GetObservable(InputElement.PointerReleasedEvent).Subscribe(async pointer =>
 				{
 					var uiConfig = Application.Current.Resources[Global.UiConfigResourceKey] as UiConfig;
-					if (!(uiConfig.Autocopy is true))
+					if (uiConfig.Autocopy is false)
 					{
 						return;
 					}
 
 					switch (MyTextBoxState)
 					{
 						case TextBoxState.AddressInsert:
-							var result = await IsThereABitcoinAddressOnTheClipboardAsync();
+							var result = await ProcessClipboardAsync();
 
 							if (result.isAddress)
 							{
-								AssociatedObject.Text = result.address;
+								AssociatedObject.Text = result.url.Address.ToString();
 							}
 							MyTextBoxState = TextBoxState.NormalTextBoxOperation;
 							var labeltextbox = AssociatedObject.Parent.FindControl<TextBox>("LabelTextBox");
```

Comment:
I would feel more comfortable if you would run an external command here which is an optional parameter of the behavior. See for example
```C#
	public class CommandOnClickBehavior : CommandBasedBehavior<InputElement>
```
And do updates of external controls there in the ViewModel. We can do a small refactor here.